system_tests/bigtable.py

```diff
@@ -93,3 +93,59 @@ def test_list_clusters(self):
             cluster_existence = (cluster in EXISTING_CLUSTERS or
                                  cluster == Config.CLUSTER)
             self.assertTrue(cluster_existence)
+
+    def test_reload(self):
+        # Use same arguments as Config.CLUSTER (created in `setUpModule`).
+        cluster = Config.CLIENT.cluster(CENTRAL_1C_ZONE, CLUSTER_ID)
+        # Make sure metadata unset before reloading.
+        cluster.display_name = None
+        cluster.serve_nodes = None
+
+        cluster.reload()
+        self.assertEqual(cluster.display_name, Config.CLUSTER.display_name)
+        self.assertEqual(cluster.serve_nodes, Config.CLUSTER.serve_nodes)
+
+    def test_create_cluster(self):
+        cluster_id = '%s-a' % (CLUSTER_ID,)
+        cluster = Config.CLIENT.cluster(CENTRAL_1C_ZONE, cluster_id)
+        operation = cluster.create()
+        # Make sure this cluster gets deleted after the test case.
+        self.clusters_to_delete.append(cluster)
+
+        # We want to make sure the operation completes.
```

Comment:
That's what I'm doing here. The protobuf long-running operations occur inside of Google and all we can do is get status updates on them. `operation.finished` actually makes an [API request][1] to get the operation's latest info.

[1]: https://github.com/GoogleCloudPlatform/gcloud-python/blob/115263bbabd40ae6cb1480e6b4cf4dea94f3f052/gcloud/bigtable/cluster.py#L173